apps/docs/content/guides/auth/phone-login/textlocal.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: 'auth-textlocal'
+title: 'Phone Auth with Textlocal'
+description: 'How to set up and use Mobile OTP with Textlocal and Supabase.'
+---
+
+<Admonition type="tip">
+
+To manage costs related to SMS authentication, consider setting project rate limits and configuring CAPTCHA. For additional cost-control strategies and to ensure compliance with TRAI's DLT requirements, visit the Production Checklist.
+
+</Admonition>
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- **A Textlocal account**: Sign up [here](https://www.textlocal.com/signup) to start sending SMS.
+- **A Supabase project**: Necessary for SMS authentication integration. Create your project [here](https://supabase.com/dashboard).
+- **A mobile phone**: To receive SMS, verifying the setup and functionality.
+- **DLT Registration**: Required for businesses in India, as per TRAI regulations, to send commercial SMS.
+
+## DLT Registration Requirements for India
+
+The Telecom Regulatory Authority of India (TRAI) mandates that all businesses intending to send commercial SMS in India must register their business entity, headers (sender names), and message templates on DLT platforms. This step is essential for legally and effectively sending SMS within India, ensuring compliance with TRAI regulations.
+
+1. **Complete your DLT registration**: Visit [this link](https://www.textlocal.in/tcccpr-trai-new-dlt-registration/) for a detailed guide on registering your business on a DLT platform.
+2. **Update your Textlocal account**: After receiving your Entity ID/Registration ID from the DLT registration, update it in your Textlocal account settings [here](https://control.textlocal.in/settings/account/).
+3. **Approval confirmation**: Share a screenshot showing your approved headers from the DLT platform with Textlocal support at support@textlocal.in to activate your SMS services.
+
+## Setting Up Textlocal as Your SMS Provider
+
+To integrate Textlocal with Supabase, you'll need:
+
+- **API Key**: For API authentication.
+- **Sender Name**: The name displayed as the SMS sender.
+- **Supabase Configuration**: Steps to integrate with your Supabase project.
+
+### Acquire Your Textlocal API Key
+
+First, log into your Textlocal account and head to the API Keys section under Settings. Here, generate a new API Key. This key will facilitate secure communication between your application and Textlocal's SMS gateway.
+
+![Get your API Keys](/docs/img/guides/auth-textlocal/2.png)
```

Comment:
I would remove this. Getting the API key from the dashboard is pretty self-explanatory without it, and screenshots age very quickly.